spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -688,11 +840,42 @@ private AnnotationNode createDataProcessorAnnotation() {
     return ann;
   }
 
+  private void createDataVariableMultiplicationsMethod() {
+    if (dataVariableMultiplications.isEmpty()) return;
```

Comment:
Is that a style in spock without braces?

Reply:
Unfortunately, yes.
A quick search for one-line `if` statements using the regex `if \(.*\).*;` yields 408 results. :-(

Reply:
When you say "Unfortunately, yes.", then why do you write it that way?

Reply:
Because worse is inconsistency imho. I usually try to adapt to the code style of the project I'm contributing to, even if I don't like it.

Reply:
I am not quite convinced, yes we should adapt to the habits and patterns of the project, so that no 10 code styles creep into the project.
But if we can't change our habits over time, IMHO "bad styles" will never die out.
This would also apply if the language gets a new feature to write something better/shorter, we would not adapt that, because the old code did not have it.

I think this code change is a good example for this:
```java
    for (DataProviderInfo provider : dataProviders)
      if (provider.getDataProviderMethod().hasBytecodeName(name)) return true;
    if (dataVariableMultiplicationsMethod != null && dataVariableMultiplicationsMethod.hasBytecodeName(name)) return true;
```
I find it pretty hard to read and at first I thought it was a formatting mistake.

Reply:
You imho cannot really compare using a new language feature with allowing one-line `if`s.
I also personally greatly prefer
```groovy
expect:
    c == 3

where:
    a | b
    1 | 2
    1 | a + 1

    c = b + 1
```
over
```groovy
expect:
c == 3
where:
a | b
1 | 2
1 | a + 1
c = b + 1
```
and 4 spaces as indentation vs. 2 spaces.

But ultimately this is a decision @leonard84 has to do.
It could also well be that in the beginning I did not use one-line `if`s and he made me change it, I don't remember. :-D

Reply:
My two cents, if anyone is interested: I do not mind one-line if's, even though I would prefer to put the action on a separate line for better debuggability. I definitely dislike braces around one-statement if/else branches, though.

Usually, I also put blank lines in between expect-where or similar, but definitely no extra indentation. And I am a proponent of two spaces, not four.

I guess, if we ask 4 people, we might get as well 5 kinds of answers. 😉